dev-libs/libpeas/libpeas-1.20.0-r1.ebuild

```diff
@@ -0,0 +1,97 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+GNOME2_LA_PUNT="yes"
+PYTHON_COMPAT=( python{3_4,3_5} )
+
+inherit autotools eutils gnome2 multilib python-single-r1 virtualx
+
+DESCRIPTION="A GObject plugins library"
+HOMEPAGE="https://developer.gnome.org/libpeas/stable/"
+
+LICENSE="LGPL-2+"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~amd64-fbsd ~amd64-linux ~x86-linux"
+
+IUSE="+gtk glade jit lua +python"
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+
+RDEPEND="
+	>=dev-libs/glib-2.38:2
+	>=dev-libs/gobject-introspection-1.39:=
+	glade? ( >=dev-util/glade-3.9.1:3.10 )
+	gtk? ( >=x11-libs/gtk+-3:3[introspection] )
+	lua? (
+		>=dev-lua/lgi-0.9.0
+		jit? ( >=dev-lang/luajit-2:2 )
+		!jit? ( =dev-lang/lua-5.1*:0 ) )
+	python? (
+		${PYTHON_DEPS}
+		>=dev-python/pygobject-3.2:3[${PYTHON_USEDEP}] )
+"
+DEPEND="${RDEPEND}
+	>=dev-util/gtk-doc-am-1.11
+	>=dev-util/intltool-0.40
+	virtual/pkgconfig
+
+	dev-libs/gobject-introspection-common
+	gnome-base/gnome-common
+"
+# eautoreconf needs gobject-introspection-common, gnome-common
+
+pkg_setup() {
+	use python && python-single-r1_pkg_setup
+}
+
+src_prepare() {
+	# Gentoo uses unversioned lua - lua.pc instad of lua5.1.pc, /usr/bin/lua instead of /usr/bin/lua5.1
+	eapply "${FILESDIR}"/${PN}-1.14.0-lua.pc.patch
+	eautoreconf
+	gnome2_src_prepare
+}
+
+src_configure() {
+	# Wtf, --disable-gcov, --enable-gcov=no, --enable-gcov, all enable gcov
+	# What do we do about gdb, valgrind, gcov, etc?
+	local myconf=(
+		$(use_enable glade glade-catalog)
+		$(use_enable gtk)
+		--disable-static
+
+		# possibly overriden below
+		--disable-python{2,3}
+		--disable-lua5.1
+		--disable-luajit
+	)
+
+	if use lua ; then
+		myconf+=( --enable-lua5.1 )
+		if use jit ; then
+			myconf+=( --enable-luajit )
+		fi
+	fi
+
+	if use python ; then
+		myconf+=(
+			--enable-python3
+			# it is just 'PYTHON' for py3 in the build system
+			PYTHON="${PYTHON}"
+			PYTHON3_CONFIG="$(python_get_PYTHON_CONFIG)"
```

Comment:
Could you check whether this is still necessary? I think python wrapping should be able to deal with it, now that we have one impl only. Then you could just put `$(use_enable python python3)` above.